db/schema.rb

```diff
@@ -41,6 +41,7 @@
     t.datetime "updated_at"
     t.integer "replaced_by_id"
     t.datetime "uploaded_to_asset_manager_at"
+    t.datetime "synchronised_with_asset_manager_at"
```

Comment:
I think this has been accidentally added and should be removed? :)

Reply:
Looks to have been removed in this commit https://github.com/alphagov/whitehall/commit/5073041d082a9766a322ec79a90973d705804d05 but instead of adding another migration to remove the field, the original migration was removed.

This may have left your database in an inconsistent state with respect to this field.

Reply:
👍